java/src/main/java/be/tothepoint/Main.java

```diff
@@ -0,0 +1,42 @@
+package be.tothepoint;
+
+import java.util.Arrays;
+import java.util.List;
+
+import be.tothepoint.application.ProjectionRunner;
+import be.tothepoint.infra.RestEventStreamProvider;
+import be.tothepoint.model.EventStreamProvider;
+import be.tothepoint.model.projections.EventTypeProjection;
+import be.tothepoint.model.projections.NrOfPlayersProjection;
+import be.tothepoint.model.projections.Projection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class Main {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(Main.class);
+
+  public static void main(String[] args) throws Exception {
+    // Switch between File based access and Remote rest here!
+    // EventStreamProvider streamProvider = new FileEventStreamProvider();
+    EventStreamProvider streamProvider = new RestEventStreamProvider();
+    List<Projection> projections = Arrays.asList(new EventTypeProjection(), new NrOfPlayersProjection());
+
+    ProjectionRunner main = new ProjectionRunner(streamProvider, projections);
```

Comment:
I'm not a huge fan of the ProjectionRunner. For me it doesn't add anything and I find it more complex to read what's going on.

I would love if the main something looked like this:

```java
List<Event> events = streamProvider.getEvents();

Projection nrOfPlayersProjection = new NrOfPlayersProjection(events);
System.out.println("Nbr of players: ");
System.out.priintln(nrOfPlayersProjection.getNrOfPlayers());
```

(The syntax is probably not completely correct)
But this would make it more easy to understand. Has les moving parts and is easier to start with.

We want it as easy as possible to start with.